### ISSUE TITLE
[Feature Request] Auto random color after separate

### ISSUE THREAD

**Issue by mamadaevv** [PEZ/SketchSeparateShapes#6]
What do you think about apply auto random color after separate?

![arndom color after separate](https://user-images.githubusercontent.com/4017693/35967989-a19da93e-0cd3-11e8-803f-7930bcd6ebc7.gif)

**Comment by mamadaevv** [comment]
It will help to better distinguish forms.

**Comment by mamadaevv** [comment]
I can create color list. To avoid this situation.

<img width="238" alt="untitled 2 2018-02-08 13-35-11" src="https://user-images.githubusercontent.com/4017693/35968449-e59fa9ba-0cd4-11e8-8807-cb37984757e0.png">

**Comment by mamadaevv** [comment]
Plugin RandomColor have a problem:
![screencast 2018-02-08 13-36-55](https://user-images.githubusercontent.com/4017693/35968588-5fc0bc02-0cd5-11e8-9c7d-55d7913bb0d9.gif)

**Comment by PEZ** [comment]
Yeah, this could be a second menu item so that the user easily can choose behaviour:
1. Separate shapes
2 Separate + colorize

Would that work? I'm thinking that the current behaviour of trying to maintain the style of the original shapes is sometimes desirable.

What is the problem you with Random Color you are showing? (Talk to me like I am five years old 😎)

**Comment by mamadaevv** [comment]
The same color for two different shapes

<img width="456" alt="screencast 2018-02-08 13-36-55 gif 2018-02-08 16-35-32" src="https://user-images.githubusercontent.com/4017693/35975561-2f9c190e-0cee-11e8-9c85-f30d448fc9d6.png">

**Comment by mamadaevv** [comment]
If you make two menu items, the magic disappears 😊. 

Also it becomes a little harder to use the plugin. 

Can we make toggle menu item? Like this:
<img width="591" alt="toggle on" src="https://user-images.githubusercontent.com/4017693/35976254-9ff6c364-0cf0-11e8-9f7e-d47abca57447.png">

But in this plugin menu item don't change title. It will be great if title will be changed.
I mean if random color is ON, title should be – Colorize [ON], 
and if color is OFF, title should be – Colorize [OFF]

I think this way is more user-friendly.

**Comment by PEZ** [comment]
I don't think we can change the menu item text dynamically, though. So the user would have to remember what the setting is, which doesn't seem very nice.

Two different options is what the Runner guidelines would point at. They would get different icons as well.

**Comment by mamadaevv** [comment]
Right! OK. 
I'll send new icon, asap.

**Comment by mamadaevv** [comment]
And two different buttons in toolbar, right?
(if this function will appear)

**Comment by PEZ** [comment]
The toolbar buttons won't happen, I think. Word has it that it is quite a big thing to fix. It might be something that future versions of Runner can assist with.

**Comment by mamadaevv** [comment]
I see
